mlrun/runtimes/sparkjob/spark3job.py

```diff
@@ -770,6 +770,11 @@ def _with_monitoring(self, enabled=True, exporter_jar=None):
                 self.spec.monitoring["exporter_jar"] = exporter_jar
 
     def with_igz_spark(self, mount_v3io_to_executor=True):
+        """
+        Configures the pods (driver and executors) to have V3IO access (via file system and via Hadoop).
+
+        :param mount_v3io_to_executor: Default=True. When False, limits the file system mount to driver pod only.
```

Comment:
We don't really use this form (`Default=True`) anywhere else in the docstrings, and it's a bit confusing because of its similarity to python syntax. We use this form in many places:
```suggestion
        :param mount_v3io_to_executor: When False, limits the file system mount to driver pod only. Default is True.
```